packages/cheatsheet/src/lib/sampleSpokenFormInfos/defaults.json

```diff
@@ -70,7 +70,7 @@
           "variations": [
             {
               "spokenForm": "pour <target>",
-              "description": "Edit new line after"
+              "description": "Edit new line/item/etc after"
```

Comment:
These are actually only used for https://www.cursorless.org/cheatsheet.  In order to show up in your local cheatsheet (`"cursorless cheatsheet"`), you'll want an override here https://github.com/cursorless-dev/cursorless/blob/b8e482572f6c3df7b6d63dd86a26dac3f1c33163/cursorless-talon/src/cheatsheet/sections/actions.py#L37

See [draft cheatsheet developer docs](https://deploy-preview-1718--cursorless.netlify.app/docs/contributing/cheatsheet/)

Reply:
The original text "Edit new line after" appears in both my local cheat sheet and the web cheat sheet, and that text only appears once in the code base.  I do not understand why I need an override.

Reply:
The description is auto-generated sentence casing the action name by default. Since the action name doesn't completely describe the action, you want to override the default description.

Reply:
Ungreppability is one of the drawbacks of too much magic 😅

Reply:
I added to actions.py but `pnpm -F cursorless-org dev` did not update the cheatsheet at http://localhost:3000/cheatsheet for me.

Reply:
There are unfortunately still a number of gotchas around cheatsheet development; one of our weaker areas for DX 😅. But if you have a look at the [deploy preview](https://deploy-preview-1772--cursorless.netlify.app/cheatsheet) you'll see that the web version has been updated fine, so you just need to update the local custom version that you get when you say `"cursorless cheatsheet"`. If you're not quite sure how to test that out, feel free to just update your PR and I can test it

We should really have docs on how to test dev version of cursorless-talon 🤔

Reply:
Btw in terms of wording, I can't say I'm completely in love with `Edit new line/item/etc after`, but I'm also not sure I have a better idea for something that gets the idea across without being too verbose for the cheatsheet. @AndreasArvidsson @josharian any ideas?

Reply:
I can't think of anything better at the moment.

Reply:
“Edit new line or other scope”?

Reply:
“Edit new line or other scope after target” has my vote.

Reply:
We're not completely consistent on whether we include the "target" at the end. I think for the most part we only say "target" when it appears in the middle of the sentence. But I could see an argument for always having "target", as we have it in the spoken form. If you do include "target", be sure to use `<target>` so it gets rendered properly

Reply:
But I'm happy with “Edit new line or other scope after”, with or without "target", so feel free to update it and be sure to update that talon side too

Reply:
@AndreasArvidsson and I just discussed in meet-up and we both like "Edit new line or other scope after", without the "target" suffix, as that's more consistent with the other descriptions.  Sound good @jmegner? If so, I think at this point just needs that update as well as updating the talon side